modular_skyrat/modules/novaya_ert/code/suit.dm

```diff
@@ -8,84 +8,6 @@
 #define NRI_HEAL_AMOUNT 10
 #define NRI_BLOOD_REPLENISHMENT 20
 
-/obj/item/clothing/suit/armor/vest/russian/nri
-	name = "\improper B42M combined armor vest"
-	desc = "A B42M combined body armor designed to protect the torso from bullets, shrapnel and blunt force. This vest performed well in the Border War against SolFed, but NRI required significant design changes due to the enemy's new and improved weaponry. These models were recently phased out and then quickly found their way onto the black market, now commonly seen in the hands (or on the bodies) of insurgents."
-	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits/armor.dmi'
-	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/suits/armor.dmi'
-	worn_icon_teshari = 'modular_skyrat/master_files/icons/mob/clothing/species/teshari/suit.dmi'
-	icon_state = "russian_green_armor"
-	armor_type = /datum/armor/russian_nri
-	supports_variations_flags = CLOTHING_NO_VARIATION
-	inhand_icon_state = "armor"
-	uses_advanced_reskins = TRUE
-	unique_reskin = list(
-		"Basic" = list(
-			RESKIN_ICON_STATE = "russian_green_armor",
-			RESKIN_WORN_ICON_STATE = "russian_green_armor"
-		),
-		"Corpsman" = list(
-			RESKIN_ICON_STATE = "russian_medic_armor",
-			RESKIN_WORN_ICON_STATE = "russian_medic_armor"
-		),
-	)
-
-/datum/armor/russian_nri
-	melee = 30
-	bullet = 40
-	laser = 20
-	energy = 30
-	bomb = 35
-	fire = 50
-	acid = 50
-	wound = 15
-
-/obj/item/clothing/suit/armor/heavy/nri
-	name = "\improper Cordun-M armor system"
-	desc = "A robust set of full-body armor designed for the harshest of environments. A modern set of heavy armor recently implemented by NRI Defense Collegium to accomodate with modern specifications. While a combination of lighter materials and a passive internal exoskeleton might assist the user's movement, you'll still be as slow as a snail."
-	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits/armor.dmi'
-	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/suits/armor.dmi'
```

Comment:
Actually I think you forgot to remove the worn icons from their dmi for the suits here. You only did the Teshari ones.

Reply:
> Actually I think you forgot to remove the worn icons from their dmi for the suits here. You only did the Teshari ones.

i forgor :skull: